tests/integration/ember-concurrency-ts-test.ts

```diff
@@ -4,7 +4,14 @@ import { setupRenderingTest } from 'ember-qunit';
 import { action, computed, set } from '@ember/object';
 import { click, render, settled } from '@ember/test-helpers';
 import { hbs } from 'ember-cli-htmlbars';
-import { Task, TaskGenerator, TaskInstance } from 'ember-concurrency';
+import {
+  Task,
+  TaskGenerator,
+  TaskInstance,
+  EncapsulatedTask,
+  EncapsulatedTaskDescriptor,
+  EncapsulatedTaskInstance,
```

Comment:
Hm, this is not public/importable right? Not sure why it worked 🤔

Reply:
Anyway, I think we just have to add the `type EncapsulatedTaskInstance<T, State extends object> = TaskInstance<T> & EncapsulatedTaskState<State>;` alias locally, until we resolve the generic arguments issue upstream and make this public, hopefully soon.

Reply:
*Everything* is importable in ambient context! :smile:
(because everything is automatically exported in ambient context)

Reply:
😞 Then we probably should make the adjustments to the upstream .d.ts file since that is definitely a bug and unintended. Can you make `EncapsulatedTaskInstance` a type alias here so we don't break it when we fix it upstream? I do think we can expose it soon, but the current version is likely missing a generic arg (in the middle).